catherine_feijoo/class2/my_calc.rb

```diff
@@ -0,0 +1,72 @@
+def addition(num_1, num_2)
+  num_1 + num_2
+end
+
+def subtraction(num_1, num_2)
+  num_1 - num_2
+end
+
+def multiplication(num_1, num_2)
+  num_1 * num_2
+end
+
+def division(num_1, num_2)
+  num_1 / num_2
+end
+
+def modulo(num_1, num_2)
+  num_1 % num_2
+end
+
+puts "what would you like to do? ('add', 'sub', 'mult', 'div', 'mod')"
+puts" [to end enter 'end']"
+
+user_oper = ""
+
+while user_oper != "end"
+  user_oper = gets.chomp
+  
+  case
+  when user_oper == "add"
+    print "enter first number  > "
+    num_1 = gets.chomp.to_f
+    print "enter second number > "
+    num_2 = gets.chomp.to_f
+    puts " > #{addition(num_1, num_2)}"
```

Comment:
In order for my `get_numbers` method to work, you would have to change line 42 to:

```
puts "> #{addition(@num_1, @num_2)}"